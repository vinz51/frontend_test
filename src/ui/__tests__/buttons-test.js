import renderer from "react-test-renderer";
import React from "react";
import Button from "../buttons";

describe("Button", () => {
  describe("Snapshots", () => {
    it("renders", () => {
      const myTreeComponent = renderer
        .create(<Button>Call me</Button>)
        .toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });

    it("renders inverted", () => {
      const myTreeComponent = renderer
        .create(<Button label="call me" inverted={true} />)
        .toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
