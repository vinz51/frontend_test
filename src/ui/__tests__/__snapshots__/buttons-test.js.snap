// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Snapshots renders 1`] = `
<button
  className="sc-AxjAm fPbywM"
>
  Call me
</button>
`;

exports[`Button Snapshots renders inverted 1`] = `
<button
  className="sc-AxjAm kppOXk"
>
  call me
</button>
`;
